spec/oca-epak/oep/client_spec.rb

```diff
@@ -33,4 +33,19 @@
       end
     end
   end
+
+  describe "#get_pdf_de_etiquetas_por_orden_or_numero_envio" do
+    let(:real_order) { 21132466 }
+    let(:fake_order) { 123_000_000 }
+    it "generates the PDF of a delivery" do
+      VCR.use_cassette("existing_delivery_pdf") do
+        result = subject.get_pdf_de_etiquetas_por_orden_or_numero_envio(
+          id_orden_retiro: real_order
+        )
+
+        expect(result).to be
+        expect(Base64.decode64(result)[0,4]).to eq("%PDF")
```

Comment:
If you have another way to evaluate that the response is a valid PDF please let me know and I'll updated it. I tried to avoid adding extra gem for testing dependencies only. But if you prefer I can add one.